components/tidb_query_expr/src/impl_time.rs

```diff
@@ -128,6 +140,48 @@ pub fn week_of_year(ctx: &mut EvalContext, t: Option<&DateTime>) -> Result<Optio
     Ok(Some(Int::from(week)))
 }
 
+#[rpn_fn(nullable, capture = [ctx])]
+#[inline]
+pub fn year_week_with_mode(
+    ctx: &mut EvalContext,
+    t: Option<&DateTime>,
+    mode: Option<&Int>,
+) -> Result<Option<Int>> {
+    if t.is_none() || mode.is_none() {
+        return Ok(None);
```

Comment:
Why not remove `nullable` instead?

Reply:
Take `week_with_mode` as example, `year_week_with_mode` is set nullable.

If it can be removed, I'll remove it.

Reply:
Yes, it can be removed.

Reply:
done

PTAL